python/paddle/fluid/contrib/slim/tests/test_imperative_qat.py

```diff
@@ -157,7 +157,8 @@ class TestImperativeQat(unittest.TestCase):
 
     def test_qat_save(self):
         imperative_qat = ImperativeQuantAware(
-            weight_quantize_type='abs_max',
+            #weight_quantize_type='abs_max',
+            weight_quantize_type='channel_wise_abs_max',
```

Comment:
Don't modify this file, please add your own unit test.

Reply:
Cool, added one new test.